compiler/record-minmax/src/RecordMinMax.cpp

```diff
@@ -32,6 +36,38 @@ using DataType = luci_interpreter::DataType;
 namespace
 {
 
+/**
+ * @brief  clipAndAverage does the following things
+ *         (1) Discard the top/bottom clip_ratio (%) values from vector
+ *             Ex: top_clip_ratio = 5, bottom_clip_ratio = 3
+ *                 discard top 5 % values and bottom 3 % values from vector
+ *         (2) Calculate the average of remaining values
+ */
+float clipAndAverage(std::vector<float> &vector, float top_clip_ratio, float bottom_clip_ratio)
+{
+  if (bottom_clip_ratio < 0 || bottom_clip_ratio >= 100)
```

Comment:
`>= 100` will not allow 100. typo? or intended?

Reply:
It was intended. If clip_ratio is 100 %, all data will be clipped, so we cannot record min/max values.